src/main/javascript/src/resources/components/footer/footer.html

```diff
@@ -0,0 +1,10 @@
+<template>
+	<div class="footer">
+		<!-- <img src="/assets/images/carleton-login-logo.svg" alt="" class="logo"/>  -->
+
+		<a href="http://carleton.ca/privacy/policies/" target="_blank">Privacy Policies</a>
+		<a href="http://carleton.ca/accessibility/" target="_blank">Accessibility</a>
+		<a href="mailto:crmsupport@carleton.ca?Subject=Support%20Request%20or%20Feedback">Support Request/Feedback</a>
+		Copyright - Carleton
+	</div>
```

Comment:
This .footer class doesn't exist. The css written is for the footer element <footer/> 
If you want to space things evenly, an example class could be
```css
.footer {
    display: flex;
    justify-content: space-between;
    align-items: center;
}
```
Each anchor tag needs to be wrapped in a div for this to take effect.
To put the copyright on the right side, you can put the text in a div or span and give it a class that has { margin-left: auto; } to push it all the way right. This will also push the other bits left though. Perhaps using { float: right; } would work as well.

Reply:
Added the .footer class and tried the <div> with the auto margin. Currently still doesn't appear to have an impact on the view.